counsel.el

```diff
@@ -5267,21 +5283,35 @@ trees."
                                 (match-string-no-properties 1)))))))
     (sort targets #'string-lessp)))
 
+(defun counsel-compile--pretty-propertize (leader text face)
+  "Return a pretty string of the form ' LEADER TEXT '
+  where LEADER is propertized with a warning face and the remaining
+  text with FACE."
+  (concat (propertize (concat " " leader " ")
+                      'face
+                      'font-lock-warning-face)
+          (propertize text 'face face)))
```

Comment:
The docstring suggests a trailing space will be added after TEXT, but the code does not agree.